spring-cloud-dataflow-docs/src/main/asciidoc/streams-skipper.adoc

```diff
@@ -96,6 +96,240 @@ $ curl http://localhost:9002/info
+    "spring.cloud.dataflow.stream.app.label": "log"
+    "spring.cloud.stream.metrics.key": "ticktock.log.${spring.cloud.application.guid}"
+    "spring.cloud.stream.bindings.input.group": "ticktock"
+    "log.name": "mylogger"
+    "spring.cloud.stream.metrics.properties": "spring.application.name,spring.application.index,spring.cloud.application.*,spring.cloud.dataflow.*"
+    "spring.cloud.dataflow.stream.name": "ticktock"
+    "spring.cloud.dataflow.stream.app.type": "sink"
+    "spring.cloud.stream.bindings.input.destination": "ticktock.time"
+  "deploymentProperties":
+    "spring.cloud.deployer.group": "ticktock"
+----
+
+In the second update, we try to add a new property for `log` application `foo2=bar2`.
+
+[source,bash]
+----
+dataflow:>stream update --name ticktock --properties app.log.foo2=bar2
+Update request has been sent for the stream 'ticktock'
+
+dataflow:>stream manifest --name ticktock
+"apiVersion": "skipper.spring.io/v1"
+"kind": "SpringCloudDeployerApplication"
+"metadata":
+  "name": "time"
+"spec":
+  "resource": "maven://org.springframework.cloud.stream.app:time-source-rabbit"
+  "resourceMetadata": "maven://org.springframework.cloud.stream.app:time-source-rabbit:jar:metadata:1.3.1.RELEASE"
+  "version": "1.3.1.RELEASE"
+  "applicationProperties":
+    "spring.metrics.export.triggers.application.includes": "integration**"
+    "spring.cloud.dataflow.stream.app.label": "time"
+    "spring.cloud.stream.metrics.key": "ticktock.time.${spring.cloud.application.guid}"
+    "spring.cloud.stream.bindings.output.producer.requiredGroups": "ticktock"
+    "spring.cloud.stream.metrics.properties": "spring.application.name,spring.application.index,spring.cloud.application.*,spring.cloud.dataflow.*"
+    "spring.cloud.stream.bindings.output.destination": "ticktock.time"
+    "spring.cloud.dataflow.stream.name": "ticktock"
+    "spring.cloud.dataflow.stream.app.type": "source"
+  "deploymentProperties":
+    "spring.cloud.deployer.group": "ticktock"
+---
+"apiVersion": "skipper.spring.io/v1"
+"kind": "SpringCloudDeployerApplication"
+"metadata":
+  "name": "log"
+"spec":
+  "resource": "maven://org.springframework.cloud.stream.app:log-sink-rabbit"
+  "resourceMetadata": "maven://org.springframework.cloud.stream.app:log-sink-rabbit:jar:metadata:1.3.1.RELEASE"
+  "version": "1.3.1.RELEASE"
+  "applicationProperties":
+    "spring.metrics.export.triggers.application.includes": "integration**"
+    "spring.cloud.dataflow.stream.app.label": "log"
+    "spring.cloud.stream.metrics.key": "ticktock.log.${spring.cloud.application.guid}"
+    "spring.cloud.stream.bindings.input.group": "ticktock"
+    "log.name": "mylogger"
+    "spring.cloud.stream.metrics.properties": "spring.application.name,spring.application.index,spring.cloud.application.*,spring.cloud.dataflow.*"
+    "spring.cloud.dataflow.stream.name": "ticktock"
+    "spring.cloud.dataflow.stream.app.type": "sink"
+    "foo2": "bar2"
```

Comment:
Perhaps we can use [Code block with callouts](https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#source-code) to make it easy to highlight the updated properties?

Reply:
Good suggestion. I will update accordingly.